docs/source/api/regions.rst

```diff
@@ -165,3 +165,63 @@ Response Structure
 		"lastUpdated": "2018-12-06 02:14:45+00",
 		"name": "Manchester"
 	}}
+
+``DELETE``
+==========
+
+.. versionadded:: 1.5
+
+Deletes a region. If no query parameter is specified, a ``400 Bad Request`` response is returned.
+
+:Auth. Required: Yes
+:Roles Required: "admin" or "operations"
+:Response Type:  Object
+
+Request Structure
+-----------------
+
+.. table:: Request Query Parameters
+
+	+-----------+----------+---------------------------------------------------------------------------------------------------------------+
+	| Name      | Required | Description                                                                                                   |
+	+===========+==========+===============================================================================================================+
+	| id        | no       | Delete :term:`Region` by integral, unique identifier                                                          |
+	+-----------+----------+---------------------------------------------------------------------------------------------------------------+
+	| name      | no       | Delete :term:`Region` by name                                                                                 |
+	+-----------+----------+---------------------------------------------------------------------------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	DELETE /api/1.5/regions?name=Manchester HTTP/1.1
+	User-Agent: curl/7.29.0
+	Host: trafficops.infra.ciab.test
+	Accept: */*
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=eyJhdXRoX2RhdGEiOiJhZG1pbiIsImV4cGlyZXMiOjE1ODEwODM3ODQsImJ5IjoidHJhZmZpY2NvbnRyb2wtZ28tdG9jb29raWUifQ--f6b59c9dde67bcd545bde3bbb4e56b33e23827e9; Path=/; Expires=Fri, 07 Feb 2020 13:56:24 GMT; Max-Age=3600; HttpOnly
```

Comment:
API examples shouldn't include actual `mojolicious` cookies, as that may or may not (though hopefully not) constitute a security risk. You can replace the value with `...` like you did above in the request example, or just omit the header entirely (I'd prefer the former, but the latter is allowed).

Reply:
Ack, I missed one. Fixed in 820659e14.